src/core/src/OmniTileset.cpp

```diff
@@ -456,7 +477,7 @@ void OmniTileset::addImageryIon(const pxr::SdfPath& imageryPath) {
     // The name passed to IonRasterOverlay needs to uniquely identify this imagery otherwise texture caching may break
     const auto uniqueName = fmt::format("imagery_ion_{}", imageryIonAssetId);
```

Comment:
I think texture caching will break if two imagery layers have different servers but the same asset it. Double check if there's anywhere else where we assume single server.

Reply:
This is potentially a major issue then. Mixing imagery layers from multiple servers is one of the core requests as I understand it.

Reply:
After looking at this some more, I think this is a remnant of `InMemoryAssetResolver` and no longer relevant. I removed the code in https://github.com/CesiumGS/cesium-omniverse/pull/578/commits/8b1c0a484e4acd9c8f3579473d4b14eaed25db7b.